x/group/spec/01_concepts.md

```diff
@@ -65,10 +64,3 @@ new votes using the `Exec` field of `Msg/CreateProposal` and `Msg/Vote` requests
 In the former case, proposers signatures are considered as yes votes.
 For now, if the proposal can't be executed, it'll still be opened for new votes and
 could be executed later on.
-
-### Changing Group Membership
```

Comment:
probably it'd be good to add some docs related to the new tallying behavior and pruning of proposals/votes